src/kaleidoscope/MatrixAddr.h

```diff
@@ -47,7 +47,7 @@ class MatrixAddr {
   constexpr MatrixAddr(uint8_t row, uint8_t col)
     : offset_(row * cols + col) {}
 
-  constexpr MatrixAddr(uint8_t offset)
+  explicit constexpr MatrixAddr(uint8_t offset)
```

Comment:
For the offset-based `MatrixAddr` constructor, we want to make sure that this is an explicit constructor, to avoid accidents.